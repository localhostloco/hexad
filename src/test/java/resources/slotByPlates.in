create_parking_lot 1
park KA-01-HH-1234 White
slot_number_for_registration_number KA-01-HH-1234