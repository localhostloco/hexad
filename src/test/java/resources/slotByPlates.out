Created a parking lot with 1 slots
Allocated slot number: 1
1

